openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -710,6 +706,45 @@ Cache-Control: no-store
 }
 ```
 
+# Nonce Endpoint {#nonce-endpoint}
+
+This endpoint allows a Client to acquire a fresh `c_nonce` value without the overhead of a full Credential Request. A Credential Issuer that requires `c_nonce` values to be incorporated into proofs in the Credential Request (see (#credential-request)) MUST offer a Nonce Endpoint.
+
+The `nonce_endpoint` Credential Issuer Metadata parameter, as defined in (#credential-issuer-parameters), contains the URL of the Credential Issuer's Nonce Endpoint.
+
+
+## Nonce Request {#nonce-request}
+
+A request for a nonce is made by sending an HTTP GET request to the URL provided in the `nonce_endpoint` Credential Issuer Metadata parameter.
+
```

Comment:
As it is, this endpoint does not uses any authentication or access control. Any HTTP client (that doesn't even need to be an OAuth client) can use it, right?
My concern is the potential for DoS, namely if a GET produces meaningful state changes (e.g. adding entries to a nonces table somewhere).

We could have the endpoint protected by the same access token used to request the credential (I didn't identify any use case where the wallet doesn't have such AT and still needs to access this endpoint).

Reply:
"Brian also clarified that the Nonce Endpoint being unprotected follows a previous WG meeting decision." - quoting you paraphrasing me :) from https://lists.openid.net/pipermail/openid-specs-digital-credentials-protocols/Week-of-Mon-20240909/000448.html